api/Services/TeamselectionService.cs

```diff
@@ -0,0 +1,16 @@
+using SpoRE.Infrastructure.Database.Teamselection;
+
+namespace SpoRE.Services;
+
+public class TeamselectionService
```

Comment:
Kan misschien helemaal weg, zie eerdere comment. Of zorg dat je hier het DB model mapt naar wat je daadwerkelijk in de FE wilt hebben.